### DISCUSSION TITLE
Extension Popup Mockup in Scratch paint editor

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#4702] [Show and tell]
![Mockup with a Menu tab containing a few buttons](https://user-images.githubusercontent.com/106490990/175427746-5ad8d420-ebc9-40f6-918e-f269d524363f.png)

I was almost going to suggest a "main menu" tab, and this is the mockup I made.

**Comment by Endermaster3690** [comment]
cloud games is already a tab
studios list?
project recorder?
what are those?

**Reply by DNin01** [reply]
Those are not part of SA now, they're just ideas I left in there to fill in the buttons.
